src/Test/FixturesAwareTestCase.php

```diff
@@ -103,6 +121,16 @@ final protected function clearElasticSearchFixtures(string $alias): self
         return $this;
     }
 
+    /**
+     * @codeCoverageIgnore
```

Comment:
Instead of adding the _@codeCoverageIgnore_ can't we add a test? Or is there any limitation that I am not aware?

For example, the test could have the following flow:
1. Load one fixture
2. Assert that the http client returns 200 for getting that fixture
3. Call the clear fixtures
4. Assert that the http client returns 500 for getting that fixture which means that the fixtures were cleared

What you think?

Reply:
Take a look now. I've splitted that test class (was growing too much) and added tests for clear fixtures